### DISCUSSION TITLE
[resolved] Numer zamówienia przy zwrocie kosztów

### DISCUSSION THREAD

**Discussion by jakubmaguza** [allegro/allegro-api#4969] [Suggestions - resolved]
Hej!
Z jakiegoś powodu aktualnie (ani na froncie, ani w API) nie pokazujecie z jakim zamówieniem powiązany jest zwrot kosztów:
![image](https://user-images.githubusercontent.com/24191650/134804926-8ad7b6a8-fbd6-40ba-972d-146d57ade205.png)

Czy moglibyście dodać taką informację?

Jeśli nie, to czy moglibyście podać listę typów, dla których nie podajecie `orderid`?

**Comment by PrzemyslawLukanowski** [comment]
Dziś wprowadziliśmy numer zamówienia dla operacji typu "zwrot kosztów" - [news](https://developer.allegro.pl/news/get-billing-billing-entries-we-have-introduced-order-id-in-a-refund-of-costs-YW19dW1VKf1).
